GVRf/Framework/jni/objects/mesh.cpp

```diff
@@ -145,6 +146,66 @@ const float* Mesh::getBoundingBoxInfo() {
     return bounding_box_info_;
 }
 
+float* Mesh::getTransformedBoundingBoxInfo(glm::mat4 *Mat) {
+
+    if (have_bounding_box_ == false)
+        getBoundingBoxInfo();
+
+    glm::mat4 M = *Mat;
+    float a, b;
+    float *transformed_bounding_box = new float[6];
+
```

Comment:
No, that's not what I suggested.

I suggested a local variable (on the stack) which is much cheaper.

On Mon, Jun 15, 2015 at 8:52 PM, soumyajitdeb <notifications@github.com>
wrote:

> In GVRf/Framework/jni/objects/mesh.cpp
> <https://github.com/Samsung/GearVRf/pull/128#discussion_r32489324>:
>
> > @@ -145,6 +146,66 @@ const float* Mesh::getBoundingBoxInfo() {
> >      return bounding_box_info_;
> >  }
> >
> > +float* Mesh::getTransformedBoundingBoxInfo(glm::mat4 *Mat) {
> > +
> > +    if (have_bounding_box_ == false)
> > +        getBoundingBoxInfo();
> > +
> > +    glm::mat4 M = *Mat;
> > +    float a, b;
> > +    float *transformed_bounding_box = new float[6];
> > +
>
> Actually that is how it was earlier. John requested me to kill the member
> variable and allocate it on the heap. So it was changed to allocating here
> and then deallocating it in the function calling it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Samsung/GearVRf/pull/128/files#r32489324>.
>